File: fly/src/main/java/com/xia/fly/integration/rxbus/IRxBusCallback.java
package com.xia.fly.integration.rxbus;

import com.blankj.rxbus.RxBusMessage;

/**
 * @author xia
 * @date 2018/8/13.
 */
public interface IRxBusCallback {

    void callback(String eventTag, RxBusMessage rxBusMessage);
}

File: fly/src/main/java/com/xia/fly/integration/rxbus/IRxBusCallback.kt
package com.xia.fly.integration.rxbus

import com.xia.flyrxbus.RxBusMessage

/**
 * @author xia
 * @date 2018/8/13.
 */
interface IRxBusCallback {

    fun callback(eventTag: String, rxBusMessage: RxBusMessage)
}
